doc/deployment_guide.md

```diff
@@ -49,5422 +49,6064 @@ Red Hat®, Red Hat Enterprise Linux®, and Ceph are trademarks or registered tra
- 
-
-3.	From your home directory, execute the sanity_test.sh script:
-
-    ```bash
-    $ cd ~/
-    $ ./pilot/deployment-validation/sanity_test.sh
-    ```
-
-4.	If you wish to clean the environment once the sanity_test.sh script has run successfully:
-
-    ```bash
-    $ cd ~/
-    $ ./pilot/deployment-validation/sanity_test.sh clean
-    ```
-
-> Note: There are deployment options [Sanity Test Settings], in the sample files which may be configured initially to run the sanity tests automatically after the overcloud deployment is successful. The relevant sanity logs will be generated in the director VM at the directory of pilot/deployment-validation/
-
-
-
-
-<div style="page-break-after: always;"></div>
-
-# Appendix D Hugepages
-
-This appendix details the guidelines for configuration of Hugepages during the deployment of Dell EMC Ready Architecture for Red Hat OpenStack Platform version 13.0. The number of Hugepages is now calculated at runtime and no longer needs to be specified by the user.
-
-> **Topics:**
-> * Hugepages Overview
-> * Prerequisites
-> * Enabling and deploying Hugepages
-> * Logging
-> * Example of sample_csp_profile.ini
-> 
-
-
-#### Hugepages overview
-
-The Dell EMC Ready Architecture for Red Hat OpenStack Platform v13.0 provides the ability to enable hugepage support on all Nova compute nodes in the solution.
-
-The Linux kernel partitions memory into basic units called pages. The default number of pages is 49152 in the x86 architecture. Hugepages allow the Linux kernel to utilize the multiple page size capabilities of modern hardware architecture.
-
-Earlier, in previous release, fixed values were used for the number of hugepages. The number of hugepages was set to 49152 and 96 when the hugepage size was set to 2MB and 1GB respectively.
-
-In Dell EMC Ready Architecture for Red Hat OpenStack Platform Software Deployment Guide - Version 13, the number of hugepages is computed during runtime. The OpenStack Ironic API is used to fetch the **total memory size** of a compute node, and the following formula is applied to calculate the number of hugepages:
-
-
-> Hugepage number = ( Total Memory Size - ( memory reserved for host OS + memory reserved for kernel)) / size of a hugepage
-
-The number of hugepages is calculated for each compute node based on that nodes memory size. The smallest number of hugepages supported by all compute nodes is then used to configure all the compute nodes with this number of hugepages. This means that all compute nodes will be configured with the number of hugepages that are supported by the compute node with the least amount of memory.
-
-#### Prerequisites
-
-The following requirements must be met before enabling hugepages:
-
-1. Memory size must be greater than or equal to 128GB on every compute node.
-
-   It is assumed that:
-   
-   - The memory reserved for the host OS = 12GB
-   - The memory reserved for the kernel = 4GB
-

```

Comment:
Need to check XFLOW if these reserved memory values still hold true for current release.